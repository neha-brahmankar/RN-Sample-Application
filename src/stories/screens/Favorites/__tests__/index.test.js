import React from "react";
import FavoritesPage from "../index";
// Note: test renderer must be required after react-native.
import renderer from "react-test-renderer";

const navigation = { state: jest.fn() };

it("renders correctly", () => {
	const tree = renderer.create(<FavoritesPage navigation={navigation} />).toJSON();
	expect(tree).toMatchSnapshot();
});
