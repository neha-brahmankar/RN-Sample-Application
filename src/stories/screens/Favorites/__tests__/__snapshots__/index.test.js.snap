// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "height": 1334,
    }
  }
  virtual={undefined}
>
  <View>
    <View
      style={
        Object {
          "backgroundColor": "#F8F8F8",
          "borderBottomColor": "#a7a6ab",
          "borderBottomWidth": 0.5,
          "elevation": 3,
          "flexDirection": "row",
          "height": 64,
          "justifyContent": "center",
          "left": 0,
          "paddingLeft": 10,
          "paddingRight": 10,
          "paddingTop": 15,
          "right": 0,
          "top": 0,
        }
      }
      virtual={undefined}
    >
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "alignSelf": "center",
            "flex": 1,
          }
        }
        virtual={undefined}
      >
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": null,
              "backgroundColor": "transparent",
              "borderBottomWidth": null,
              "borderColor": null,
              "borderLeftWidth": null,
              "borderRadius": 5,
              "borderRightWidth": null,
              "borderTopWidth": null,
              "elevation": 0,
              "flexDirection": "row",
              "height": 45,
              "justifyContent": "space-between",
              "marginLeft": -3,
              "opacity": 1,
              "paddingBottom": 6,
              "paddingLeft": 6,
              "paddingRight": 6,
              "paddingTop": 6,
              "shadowColor": null,
              "shadowOffset": null,
              "shadowOpacity": null,
              "shadowRadius": null,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Text
            accessible={true}
            active={true}
            allowFontScaling={false}
            disabled={false}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 12,
                },
                Array [
                  Object {
                    "color": "#000",
                    "fontSize": 30,
                  },
                  Object {
                    "color": "#007aff",
                    "fontSize": 33,
                    "marginLeft": 2,
                    "marginRight": 2,
                    "marginTop": 2,
                    "paddingTop": 2,
                  },
                ],
                Object {
                  "fontFamily": "Ionicons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
            virtual={undefined}
          >
            ï‡ƒ
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "flex": 1,
          }
        }
        virtual={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "#000",
              "fontFamily": "System",
              "fontSize": 17,
              "fontWeight": "600",
              "textAlign": "center",
            }
          }
          virtual={undefined}
        >
          My Favorites
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "flex-end",
            "alignSelf": "center",
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "flex-end",
          }
        }
        virtual={undefined}
      />
    </View>
  </View>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutoAutomaticScroll={true}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    onScroll={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={0}
    showsVerticalScrollIndicator={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
    virtual={undefined}
  >
    <View>
      <View
        closeOnRowBeginSwipe={false}
        closeOnRowPress={true}
        closeOnScroll={true}
        directionalDistanceChangeThreshold={2}
        disableLeftSwipe={false}
        disableRightSwipe={false}
        leftOpenValue={0}
        previewFirstRow={false}
        recalculateHiddenLayout={false}
        rightOpenValue={0}
        style={Object {}}
        swipeToOpenPercent={50}
        virtual={undefined}
      >
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
            ]
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <View
            avatar={true}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderBottomWidth": null,
                "borderColor": "#c9c9c9",
                "flexDirection": "row",
                "marginLeft": 15,
                "paddingBottom": null,
                "paddingRight": null,
                "paddingTop": null,
              }
            }
            virtual={undefined}
          >
            <View
              style={
                Object {
                  "alignItems": "flex-start",
                  "alignSelf": "center",
                  "flex": 0,
                  "flexDirection": "row",
                }
              }
              virtual={undefined}
            >
              <Image
                source={
                  Object {
                    "uri": "http://static.food2fork.com/Jalapeno2BPopper2BGrilled2BCheese2BSandwich2B12B500fd186186.jpg",
                  }
                }
                style={
                  Object {
                    "borderRadius": 28,
                    "height": 56,
                    "width": 56,
                  }
                }
                virtual={undefined}
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": null,
                  "alignSelf": null,
                  "borderBottomWidth": 0.5,
                  "borderColor": "#c9c9c9",
                  "flex": 1,
                  "marginLeft": 15,
                  "paddingBottom": 10,
                  "paddingTop": 10,
                }
              }
              virtual={undefined}
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                    "marginLeft": null,
                    "marginRight": 10,
                  }
                }
                uppercase={false}
                virtual={undefined}
              >
                Kumar Pratik
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                note={true}
                style={
                  Object {
                    "color": "#808080",
                    "fontFamily": "System",
                    "fontSize": 14,
                    "fontWeight": "200",
                    "marginLeft": null,
                    "marginRight": 10,
                  }
                }
                uppercase={false}
                virtual={undefined}
              >
                Doing what you like will always keep you happy . .
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "flex-end",
                  "alignSelf": "stretch",
                  "borderBottomWidth": 0.5,
                  "borderColor": "#c9c9c9",
                  "flex": 0,
                  "paddingBottom": 10,
                  "paddingLeft": null,
                  "paddingRight": 15,
                  "paddingTop": 10,
                }
              }
              virtual={undefined}
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                note={true}
                style={
                  Object {
                    "alignSelf": null,
                    "color": "#808080",
                    "fontFamily": "System",
                    "fontSize": 12,
                    "fontWeight": "200",
                  }
                }
                uppercase={false}
                virtual={undefined}
              >
                3:43 pm
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
